src/plugins/grass/qgsgrassregion.cpp

```diff
@@ -25,6 +25,13 @@
 #include "qgsmaptool.h"
 #include "qgsexception.h"
 
+
+#include <QtGlobal>
+#if QT_VERSION >= QT_VERSION_CHECK(5, 11, 0)
```

Comment:
for these ones, why not just always include them? (since there's no real harm in unneeded includes)

Reply:
this conditional reminds us of the reason why they are included

Reply:
In what situation will this reminder be useful?